src/index.html

```diff
@@ -28,101 +28,101 @@ <h2>Filter</h2>
         <div class="filters">
           <div class="single-filter">
             <label for="antipasti">
-              <input type="checkbox" id="antipasti" name="antipasti" value="yes">Antipasti
+              <input class="filter-checkbox tag" type="checkbox" id="antipasti" value="yes">Antipasti
             </label>
           </div>
           <div class="single-filter">
             <label for="antipasto">
-              <input type="checkbox" id="antipasto" name="antipasto" value="yes">Antipasto
+              <input class="filter-checkbox tag" type="checkbox" id="antipasto" value="yes">Antipasto
```

Comment:
curious if we use both of these classes or just 'tag'. We could consider deleting one if it is not getting used